### DISCUSSION TITLE
Change the Image Number

### DISCUSSION THREAD

**Discussion by agarzon** [lllyasviel/Fooocus#307] [General]
It is possible to defined the default **Image Number** (currently: 2) to any number from a config file or as launch argument? 

Actually would be nice to be able to have some negative prompt pre-saved somewhere too.

![image](https://github.com/lllyasviel/Fooocus/assets/339828/e5397c96-f93a-4d83-9f71-b9c6bf1f4ac2)

**Comment by runew0lf** [comment]
I have a fork with a lot of these options added while still maintaining the originality - https://github.com/runew0lf/RuinedFooocus/tree/main

**Reply by agarzon** [reply]
Looks promising, I hope you can keep it updated from the main repo. 👍 I'll give it a try.
